src/test/kotlin/com/sourcegraph/cody/autocomplete/action/TriggerAutocompleteActionTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.sourcegraph.cody.autocomplete.action
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
```

Comment:
wow, I was not aware of these 😮

Reply:
You mean BasePlatformTestCase? Honestly, I am very surprised. :smile: These fixtures are the most default way to test plugin functionalities. 

In general, I recommend everything that inherits from UsefulTestCase.